python/ray/dashboard/dashboard.py

```diff
@@ -116,7 +117,7 @@ def __init__(self,
 
         self.node_stats = NodeStats(redis_address, redis_password)
         self.raylet_stats = RayletStats(redis_address, redis_password)
-        self.tune_stats = TuneCollector(None, logger, 2.0)
+        self.tune_stats = TuneCollector(DEFAULT_RESULTS_DIR, logger, 2.0)
```

Comment:
How hard would it be to enable this to be a variable?

Reply:
I think `logger` is a global variable, you don't need to pass it in to the TuneCollector constructor.